### DISCUSSION TITLE
Spinner with designated speed

### DISCUSSION THREAD

**Discussion by LIII-Works** [ppy/osu#13210] [Ideas]
### Describe the new feature:
Allow the mapper to specify how fast a spinner needs to be turned or decide according to BPM. Spinner score is calculated based on how long the player can get close to this value. In another word, the better player can sync with the speed mapper set, the more score will be awarded, vice versa.

**Comment by LittleEndu** [comment]
Unless there's an unambiguous way to show how fast you need to spin the spinner, this idea just isn't going to work. 

You could show the target spins per minute that the mapper set but then you run into the issue that humans tend to think in logarithmic scale, so just a number saying "200 spm" vs a number saying "300 spm" isn't going to be unambiguous enough in my opinion. 

Also this is ignoring the fact that spinners are already rare hitobjects and spinner bonus already controversial topic.

**Comment by LIII-Works** [comment]
Can't disagree with your opinion, maybe make the speed meter like hit error, put a mark on a bar to show the designated speed  and a marker moving left or right to show how fast the player is spinning
